Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -230,7 +230,6 @@ - (void)createSimulatorWithContext:(BPExecutionContext *)context {
     };
 
     handler.onSuccess = ^{
-        context.simulatorCreated = YES;
```

Comment:
As far as I can tell, the actual creation of the device is successful if the device is set to a non-nil value here: https://github.com/linkedin/bluepill/blob/master/Bluepill-cli/Bluepill-cli/Simulator/BPSimulator.m#L61. As I mentioned above, however, `createSimulator` really does more things that can fail along the way.

I think checking if `context.runner.device` is `nil` is safer than using this property anyway.